tests/mbt/model/ccv_model.qnt

```diff
@@ -229,6 +238,17 @@ module ccv_model {
     // UTILITY FUNCTIONS
     // ==================
 
+    pure def removeZeroPowers(valSet: Node -> int): Node -> int =
```

Comment:
Isn't possible to have this as `removeZeroPowers(valSet: ValidatorSet): ValidatorSet`?